### DISCUSSION TITLE
v1.1.0-beta.2

### DISCUSSION THREAD

**Discussion by EvanMulawski** [EvanMulawski/FanControl.CorsairLink#23] [Announcements]
- Internals: Handles the possibility of a Corsair device not supporting the query for its product name.
- Logging: Logs the device firmware version when the plugin loads.

<hr /><em>This discussion was created from the release <a href='https://github.com/EvanMulawski/FanControl.CorsairLink/releases/tag/v1.1.0-beta.2'>v1.1.0-beta.2</a>.</em>

**Comment by Rockstar04** [comment]
Thanks for the work on this really excited to try to get this working. Just pulled the beta.2 dll but I am not able to see my H115i Platinum.

FanControl version: 148
iQue version 4.33.138

Here is my latest logs.txt and USBDeview output.
```
3/7/2023 6:24:18 PM: [CorsairLink] DeviceManager: Found 5 Corsair device(s)
  name=, devicePath=\\?\hid#virtualdevice&10&col02#2&28347c50&0&0001#{4d1e55b2-f16f-11cf-88cb-001111000030}\kbd
  name=, devicePath=\\?\hid#virtualdevice&10&col03#2&28347c50&0&0002#{4d1e55b2-f16f-11cf-88cb-001111000030}\kbd
  name=, devicePath=\\?\hid#virtualdevice&10&col01#2&28347c50&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}
  name=, devicePath=\\?\hid#virtualdevice&10&col05#2&28347c50&0&0004#{4d1e55b2-f16f-11cf-88cb-001111000030}
  name=, devicePath=\\?\hid#virtualdevice&10&col04#2&28347c50&0&0003#{4d1e55b2-f16f-11cf-88cb-001111000030}

3/7/2023 6:24:18 PM: [CorsairLink] DeviceManager: Found 0 supported Corsair device(s)

3/7/2023 6:24:18 PM: CorsairLink could not initialize or has no sensors.
```

```
H115i Platinum	Corsair H115i Pro	Vendor Specific	Yes	Yes	No	No	7289_2.0	3/7/2023 6:08:27 PM	7/7/2021 7:28:44 PM	1b1c	0c13	1.00	ff	00	00	DESKTOP-TRENT			SIUSBXP		SiUSBXp.sys		Corsair Components, Inc.	@oem29.inf,%FRIENDLYNAME%;Corsair USBXp Driver	64 mA	2.00	Corsair H115i Pro	3.3.0.0	DriverInstall.NT	oem29.inf	USB\VID_1B1C&PID_0C13\7289_2.0	Removable, UniqueID, SurpriseRemovalOK					

```

**Reply by EvanMulawski** [reply]
Hi @Rockstar04, unfortunately that device (`0c13`) is not supported because its USB class is not HID. See also: https://github.com/EvanMulawski/FanControl.CorsairLink/discussions/15#discussioncomment-4950855
